atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTestSuite.java

```diff
@@ -57,56 +56,46 @@
         SchemaMutationLockTablesIntegrationTest.class
         })
 public final class CassandraTestSuite {
-    public static final int THRIFT_PORT_NUMBER = 9160;
-
-    private static final DockerComposeRule docker = DockerComposeRule.builder()
+    private static final DockerComposeRule DOCKER = DockerComposeRule.builder()
             .file("src/test/resources/docker-compose.yml")
-            .waitingForHostNetworkedPort(THRIFT_PORT_NUMBER, toBeOpen())
-            .saveLogsTo("container-logs")
+            .saveLogsTo(LogDirectory.circleAwareLogDirectory(CassandraTestSuite.class))
             .build();
+    private static final DockerProxyRule DOCKER_PROXY_RULE = new DockerProxyRule(
+            DOCKER.projectName(),
+            CassandraTestSuite.class);
     private static final Gradle GRADLE_PREPARE_TASK =
             Gradle.ensureTaskHasRun(":atlasdb-ete-test-utils:buildCassandraImage");
 
     @ClassRule
     public static final RuleChain CASSANDRA_DOCKER_SET_UP = RuleChain.outerRule(GRADLE_PREPARE_TASK)
-            .around(docker);
-
-    static InetSocketAddress cassandraThriftAddress;
+            .around(DOCKER)
+            .around(DOCKER_PROXY_RULE);
 
-    static ImmutableCassandraKeyValueServiceConfig cassandraKvsConfig;
+    static final ImmutableCassandraKeyValueServiceConfig KVS_CONFIG = ImmutableCassandraKeyValueServiceConfig.builder()
```

Comment:
Yes, you are correct. I forgot to change that. Do you want to do it in your PR when you get merge conflicts? ^^